src/main/webapp/js/feed-loader.js

```diff
@@ -0,0 +1,50 @@
+// Fetch messages and add them to the page.
+function fetchMessages() {
+    const url = '/feed';
+    fetch(url).then((response) => {
+        return response.json();
+    }).then((messages) => {
+        const messageContainer = document.getElementById('message-container');
+        if(messages.length == 0) {
+            messageContainer.innerHTML = '<p>There are no posts yet.</p>';
+        }
+        else {
+            messageContainer.innerHTML = '';
+        }
```

Comment:
This can be made a bit more concise using a ternary expression:
```suggestion
        messageContainer.innerHTML  = messages.length == 0 ? '<p>There are no posts yet.</p>': '';
```